test/ttmlir/Conversion/StableHLOToTTIR/concat_op.mlir

```diff
@@ -0,0 +1,12 @@
+// REQUIRES: stablehlo
+// RUN: ttmlir-opt --stablehlo-to-ttir-pipeline %s | FileCheck %s
+module @jit_concat attributes {} {
+  func.func public @test_concat(%arg0: tensor<32x32xf32>, %arg1: tensor<32x64xf32>) -> tensor<32x96xf32> {
+    %0 = "stablehlo.concatenate"(%arg0, %arg1) {
+    dimension = 1 : i64
+    } : (tensor<32x32xf32>, tensor<32x64xf32>) -> tensor<32x96xf32>
```

Comment:
Can you please add more cases to this test, for example different variations of concatenate? 
e.g. is the following supported?
// %input0: [[1, 2], [3, 4], [5, 6]]
// %input1: [[7, 8]]
%result = "stablehlo.concatenate"(%input0, %input1) {
  dimension = 0 : i64
} : (tensor<3x2xi64>, tensor<1x2xi64>) -> tensor<4x2xi64>
// %result: [[1, 2], [3, 4], [5, 6], [7, 8]]